VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts

```diff
@@ -343,4 +352,8 @@ export class CommandCentreComponent implements OnInit, OnDestroy {
             return false;
         }
     }
+
+    handleAllocationUpdate(allocationHearingMessage: NewAllocationMessage) {
```

Comment:
Do we need to update the list when an allocation has been made?

Reply:
I don't understand your question. We need to handle the message when a an allocation has been made

Reply:
When the client receives a notification they have more hearings allocated to them, does the list on the left hand side of the screen need to be refreshed to show an updated list?

Reply:
no it does not need to update the list